### DISCUSSION TITLE
Graphics logos

### DISCUSSION THREAD

**Discussion by stalampas** [blueseerERP/blueseer#100] [Q&A] [answered]
Can we change the apps background and logos
![Screenshot_20230212_122652](https://user-images.githubusercontent.com/125166382/218288172-91f99547-f757-46f4-b903-1bf8fe37190e.png)

**Comment by vaughnte** [comment, marked as answer]
You cannot change the background logo on the main screen....it's locked into a method in the parent frame (the image rotation method).   You can, however, change the background color using navcode "sysc".   You can set the R,G,B values to whatever background color you want.  You can also set the logo on some of the report prints (invoice, order, etc) by changing the site logo in site master.
